CHANGELOG.md

```diff
@@ -100,7 +100,6 @@ All notable changes to this project are documented in this file.
  - Skia renderer: Improve rendering quality of layers
  - GridLayout: Fixed panic when rowspan or colspan is 0 (#6181)
 
-
```

Comment:
Please re-add this line. (Unfortunately GitHub doesn't support that as a suggestion in the code-review, otherwise I'd do it. I can also do it by hand by pushing into your branch, if you prefer not to do it).